src/components/shared/Footer/FootStaticLinks.tsx

```diff
@@ -32,7 +32,7 @@ const Link2 = [
     isExternal: false,
   },
   {
-    title: '更新情報（Twitter）',
+    title: '更新情報',
```

Comment:
```suggestion
    title: '更新情報（X）',
```
外部媒体に遷移することがリンクを踏む前にわかったほうが誠実な感じがするので、情報としては残しておけたら！

Reply:
取り込みます！